docs/docs/faq.md

```diff
@@ -77,6 +77,10 @@ Therefore in ZMK, [board](#what-is-a-board) and [shield](#what-is-a-shield) are
 
 Please note, many keyboards only have a single PCB which includes the “brains” (MCU) onboard. In ZMK, these have no [shield](#what-is-a-shield), only a [board](#what-is-a-board).
 
+### Does ZMK Support Wired Split?
+
+Currently, ZMK only supports wireless split, but wired split is planned and open to contributions!
```

Comment:
It is?  It's possible sure, but I wasn't aware that it's been planned?

Reply:
I believe @petejohanson has mentioned a few times in the discord that anyone can start contributing to wired split. I am planning on changing the wording to make it clear no one has started working on it yet.

Reply:
Thanks!  Yep, there's a fine line of distinction there.  We're open to someone contributing wired split, but that doesn't mean it's high on our own TODO lists at the moment.